index.html

```diff
@@ -50,6 +50,110 @@ <h1 >Hey There.
         </section>
+        <section class="works">
+            <h1>My Recent Works</h1>
+            <hr>
+            <div class="grid works-list">
+                <article class="card work">
+                    <div class="background">
+                        <img src="https://picsum.photos/200/300" alt="">
+                    </div>
+                    <div class="content">
+                        <h2>Lorem ipsum</h2>
+                        <p>
+                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorum necessitatibus
+                            voluptatum et facere illum aliquam nulla distinctio quidem nostrum quas iusto, 
+                            obcaecati sequi, aliquid quos at reprehenderit deleniti eius. Necessitatibus.
+                        </p>
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <button class="live">See Project</button>
+                </article>
+                <article class="card work">
+                    <div class="background">
+                        <img src="https://picsum.photos/200/300" alt="">
+                    </div>
+                    <div class="content">
+                        <h2>Lorem ipsum</h2>
+                        <p>
+                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorum necessitatibus
+                            voluptatum et facere illum aliquam nulla distinctio quidem nostrum quas iusto, 
+                            obcaecati sequi, aliquid quos at reprehenderit deleniti eius. Necessitatibus.
+                        </p>
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <button class="live">See Project</button>
+                </article>
+                <article class="card work">
+                    <div class="background">
+                        <img src="https://picsum.photos/200/300" alt="">
+                    </div>
+                    <div class="content">
+                        <h2>Lorem ipsum</h2>
+                        <p>
+                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorum necessitatibus
+                            voluptatum et facere illum aliquam nulla distinctio quidem nostrum quas iusto, 
+                            obcaecati sequi, aliquid quos at reprehenderit deleniti eius. Necessitatibus.
+                        </p>
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
```

Comment:
[OPTIONAL]
- Let's remove the margin from the left to make your design better. What do you think? see the screenshot I made for your design below:

<img width="197" alt="image" src="https://user-images.githubusercontent.com/77190203/148591416-3650dfcc-de36-48d3-ab3c-d7756b15170a.png">